src/aat/resources/features/F-047/S-575.td.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "must return 200 and a list of case ids a user has access to",
+  "_guid_": "S-575",
```

Comment:
Title is wrong. This is a  negative scenario.  Please update as per feature file.

Reply:
done